web/app/controllers/drivingSessionCtrl.js

```diff
@@ -1,85 +1,91 @@
 var DrivingSession = require("../models/DrivingSession");
 var User = require("../models/User");
 
-//TODO: find an accurate formula for distance from gps points
+// TODO: find an accurate formula for distance from gps points
+// TODO 2: More likely, use Google maps, depends on what mobile is doing and
+// what we can expect the data to look like
 function calculateDistance(drivePoints) {
-  // 0 for now
-  var distance = 0;
-  // drivePoints obj has lat, lon, speed, time
-  return distance;
+	// 0 for now
+	var distance = 0;
+	// drivePoints obj has lat, lon, speed, time
+	return distance;
 }
 
+// Diff in time in Milliseconds
+// Takes in two MomentJS objects
 function calculateDuration(startTime, endTime) {
-  // Calculate duration from times in date
-  var duration = 0;
-  return duration;
+	return endTime.diff(startTime);
 }
 
 function updateStudentDrive(userId, newDrivingSession, res) {
-  User.findOne({ _id: userId }, function(err, student) {
-    if(!err) {
-      student.drivingSessions.push(newDrivingSession);
-      student.save(function(saveErr) {
-        if(saveErr){
-          res.send("Error saving driving session to student, " + saveErr);
-        }
-      });
-    } else {
-      res.send("Error retrieving driving session data, " + err);
-    }
-  });
+	User.findOne({
```

Comment:
So this was changed structure/organization wise to make the code easier to read?